src/pages/index.js

```diff
@@ -174,15 +170,18 @@ function handleAddCardFormSubmit(cardData) {
     });
 }
 
-function handleProfileImageForm() {
+function handleProfileImageForm(inputValues) {
   //evt.preventDefault();
+  console.log(inputValues);
   profileImageModal.setLoading(true);
   const profileImage = document.querySelector(".profile__image");
   const profileImageInput = document.querySelector("#profile-image-link");
   profileImage.src = profileImageInput.value;
```

Comment:
Please, delete it

Only method `_getInputValues` collects all the data of the inputs and passes it to the call of `submitForm`. So you need to use the collected data rather than the inputs.